### PR TITLE
Exercise1

### PR DESCRIPTION
**Pull request by Haribiddacdw**

### CODE REVIEW

File: Exercise_1/day1/Program10.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package day1;
+
+public class Program10 {
+	public static void main(String[] args) {
+
```
Comment (Naveen-nk-cdw) [inline Exercise_1/day1/Program10.java:15]: Get the input from the user instead of initialising

File: Exercise_1/day1/Program10.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+				            {9,10,11,12},
+				            {13,14,15,16}
+				           };
+		  System.out.println("Two Dimensional Matrix :");
+		  print(array);
+
+	  }
+	  public static void print(int array[][])
+	  {
+
```
Comment (Naveen-nk-cdw) [inline Exercise_1/day1/Program10.java:25]: please remove extra empty lines in files and keep the intendation same for every files

File: Exercise_1/day1/Program11.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+public class Program11 {
+	public static void main(String[] args) {
+
+		  int n = 3;
+		  System.out.println("Pattern using Two Dimensional Matrix :");
+		  print(n);
+
+	  }
+	  public static void print(int n)
```
Comment (Naveen-nk-cdw) [inline Exercise_1/day1/Program11.java]: ensure proper intendations for readability

File: Exercise_1/day1/Program13.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+			 {
+				 marks[i][j]=scan.nextInt();
+			 }
+		 }
+		 print(marks,student,no_sub);
+		 
+	}
+     
+	  }
+
```
Comment (Naveen-nk-cdw) [inline Exercise_1/day1/Program13.java]: name the variables in cameCase

File: Exercise_1/day1/Program13.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+		  {
+			  int sum =0;
+			  
+			  for(int j=1;j<=no_sub;j++)
+			  {
+				  
+					sum=sum+ array[i][j];
+				  
+			  }
+			  System.out.println ("the mark of student "+(i+1)+" is "+sum);
```
Comment (Naveen-nk-cdw) [inline Exercise_1/day1/Program13.java]: remove extra println as one is sufficient to take the console to pretty output

File: Exercise_1/day1/Program15.java [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+	  System.out.println();
+	  System.out.println("check wether given number is Armstrong number , Palindrone, Perfect Number ");
+	  System.out.println();
+	  pA.armstrongNumber(number);
+	  pA.palindromeNumber(number);
+	  pA.perfectNumber(number);
+  }
+  static void primeNumber()
+  {
+	  System.out.println();
```
Comment (Naveen-nk-cdw) [inline Exercise_1/day1/Program15.java]: name the variables more meaning fully instead of using single letters

**Review by Naveen-nk-cdw — CHANGES_REQUESTED** [review]
Please go through the basic review comments and change it for all files and we can have the logic and implementation review over a call

File: Exercise_1/day1/Program10.java (hunk 1 above)
Comment (Haribiddacdw) [reply Exercise_1/day1/Program10.java:15]: Updated

File: Exercise_1/day1/Program10.java (hunk 2 above)
Comment (Haribiddacdw) [reply Exercise_1/day1/Program10.java:25]: updated

File: Exercise_1/day1/Program11.java (hunk 3 above)
Comment (Haribiddacdw) [reply Exercise_1/day1/Program11.java]: noted

File: Exercise_1/day1/Program13.java (hunk 5 above)
Comment (Haribiddacdw) [reply Exercise_1/day1/Program13.java]: Updated

File: Exercise_1/day1/Program15.java (hunk 6 above)
Comment (Haribiddacdw) [reply Exercise_1/day1/Program15.java]: Updated

File: Exercise_1/day1/Program13.java (hunk 4 above)
Comment (Haribiddacdw) [reply Exercise_1/day1/Program13.java]: Updated

**Pull request merged**